coredns/endpointslice/map.go

```diff
@@ -111,6 +119,35 @@ func (m *Map) Put(es *discovery.EndpointSlice) {
 		return
 	}
 
+	// TODO: Only do this look up if globalnet enabled to reduce unnessary call to Kube API
```

Comment:
Can you create an issue to track this TODO item?

Reply:
Created #775